student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'Unknown', _parent_permission: true)
+    super(age, name)
```

Comment:
- [ ] parent_permission should be included as an argument when calling super as well.